guides/RestAPIGuide.md

```diff
@@ -1072,203 +1072,203 @@ The most common attachment type is 'file'.
 }
 ```
 **Link Attachment**
-```
+```json
 {
   * "type" : "url",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "url" : <string>,
-  "disabled" : <boolean>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "url" : "<string>",
+  "disabled" : "<boolean>",
```

Comment:
Shouldn't need to quote booleans.

Reply:
https://github.com/equella/equella.github.io/pull/25#discussion_r234487858